docker/DockerfileUIProd

```diff
@@ -2,18 +2,21 @@ FROM node:18.20.4
 
 WORKDIR /app
 
+
 # install JavaScript dependencies
 COPY src/frontend/package*.json ./
 RUN npm install
 
 # get MicroPowerManager source code and configuration
 COPY src/frontend .
 
-# build app for production with minification
-RUN npm run build
+
+COPY docker/entrypoint-ui-prod.sh /usr/local/bin/
+RUN chmod +x /usr/local/bin/entrypoint-ui-prod.sh
+ENTRYPOINT ["/usr/local/bin/entrypoint-ui-prod.sh"]
+
 
 # serve the application
```

Comment:
Can you update the comment? It's no longer correct. Here we are just installing the webserver not actually serving the application.